### DISCUSSION TITLE
Feature: Graphs!

### DISCUSSION THREAD

**Discussion by emmadesilva** [Astrotomic/opendor.me#161] [Ideas]
Would be really cool to get some graphs integrated into the profiles!

**Comment by Gummibeer** [comment]
Hey, what kind of graphs do you have in mind? Which information do you want to show?

**Reply by emmadesilva** [reply]
I was thinking graphs and counters in general. Stuff like contribution counts/graphs, lines of code written. I like data! Just seeing the repositories I've contributed to doesn't say much IMO
